KNOWN_ISSUES.md

```diff
@@ -120,10 +120,12 @@ consult the Cloudant documentation for further information.
 
 ### Session authentication
 
-Session authentication does not work with compressed request bodies when the server is CouchDB <= 3.1.1 or Cloudant <= 8169. Disabling gzip compression for requests is required when using session authentication with these server versions:
+Session authentication does not work with compressed request bodies when the server is CouchDB <= 3.1.1 or Cloudant <= 8169. [Disabling gzip compression](#disabling-request-body-compression) for requests is required when using session authentication with these server versions
```

Comment:
This bug is not corrected yet in Cloudant, however the current version is: `8192`.
Secondly, this text part and also the reference itself would be better place in the `sdks` repo.